Frameworks/UIKit/UIPickerView.mm

```diff
@@ -357,7 +363,7 @@ - (int)_selectedRow {
 }
 
 - (id)_cellSelectedUp:(id)cell {
```

Comment:
even better; thanks!

---
In reply to: [95006609](https://github.com/Microsoft/WinObjC/pull/1643#discussion_r95006609) [](ancestors = 95006609)